sefaria/helper/schema.py

```diff
@@ -42,43 +42,49 @@ def handle_dependant_indices(title):
         record.save()
 
 
-def insert_last_child(new_node, parent_node):
-    return attach_branch(new_node, parent_node, len(parent_node.children))
+def insert_last_child(new_node, parent_node, invalid=False):
+    return attach_branch(new_node, parent_node, len(parent_node.children), invalid)
 
 
-def insert_first_child(new_node, parent_node):
-    return attach_branch(new_node, parent_node, 0)
+def insert_first_child(new_node, parent_node, invalid=False):
```

Comment:
Let's use a more descriptive name for `invalid` - perhaps `skip dependencies` or `skip library rebuild`